<?xml version="1.0" encoding="UTF-8"?>
<!-- Generated by org.testng.reporters.JUnitXMLReporter -->
<testsuite ignored="4" hostname="nb-noh" failures="1" tests="5" name="ValidationTask" time="61.905" errors="0" timestamp="2022-08-22T19:35:22 PKT">
  <testcase classname="ValidateCart" name="womenCategories" time="26.574"/>
  <testcase classname="ValidateCart" name="menCategories" time="11.79"/>
  <testcase classname="ValidateShoppingBag" name="goToShopping" time="5.318"/>
  <testcase classname="ValidateRemoveCart" name="removeCart" time="1.905"/>
  <testcase classname="ValidateHomeNavigation" name="home" time="3.285">
    <failure type="java.lang.AssertionError" message="expected [https://www.tamanna.com/] but found [https://www.tamanna.com/men/overview]">
      <![CDATA[java.lang.AssertionError: expected [https://www.tamanna.com/] but found [https://www.tamanna.com/men/overview]
at ValidateHomeNavigation.home(ValidateHomeNavigation.java:21)
at java.base/java.util.ArrayList.forEach(ArrayList.java:1511)
at com.intellij.rt.testng.IDEARemoteTestNG.run(IDEARemoteTestNG.java:66)
at com.intellij.rt.testng.RemoteTestNGStarter.main(RemoteTestNGStarter.java:109)
... Removed 28 stack frames]]>
    </failure>
  </testcase> <!-- home -->
  <testcase classname="BaseClass" name="@BeforeTest navigation" time="0.0">
    <ignored/>
  </testcase> <!-- @BeforeTest navigation -->
  <testcase classname="BaseClass" name="@BeforeTest navigation" time="0.0">
    <ignored/>
  </testcase> <!-- @BeforeTest navigation -->
  <testcase classname="BaseClass" name="@BeforeTest navigation" time="0.0">
    <ignored/>
  </testcase> <!-- @BeforeTest navigation -->
  <testcase classname="BaseClass" name="@BeforeTest navigation" time="0.0">
    <ignored/>
  </testcase> <!-- @BeforeTest navigation -->
</testsuite> <!-- ValidationTask -->
